HISTORY.rst

```diff
@@ -5,6 +5,9 @@ History
 Unreleased / main
 -------------------
 
+* New method ``update_dc_append_comments`` available in the ``Acquisition`` and ``MXAcquisition`` classes
+* Update SQLAlchemy ORM models for ispyb-database v1.34.1
+
 7.1.0 (2023-01-18)
 -------------------
 * Update SQLAlchemy ORM models for ispyb-database v1.34.0
```

Comment:
Technically the ORM models only updated to 1.34.0 so far, but I don't think the changes in 1.34.1 would affect the ORM models (but the auto-ORM update _should_ trigger a PR if they do).
```suggestion
* New method ``update_dc_append_comments`` available in the ``Acquisition`` and ``MXAcquisition`` classes
* Update SQLAlchemy ORM models for ispyb-database v1.34.0
```

Reply:
Good point! Fixed!